### PR TITLE
Linking corresponding vote/ticket for a ticket/vote

### PR DESCRIPTION
**Pull request by chappjc**
For discussion on @McEdward's branch

### CODE REVIEW

File: views/tx.tmpl [hunk 1]
```diff
@@ -32,6 +32,28 @@
...
+                        {{if .Txid}}
+                            <tr>
+                                <td width="90" class="text-right pr-2 h1rem p03rem0 xs-w91">TICKET SPENT</td>
+                                <td><a href="/explorer/tx/{{.Txid}}">{{.Txid}}</a></td>
+                            </tr>
+                        {{end}}
+                    {{end}}
+                {{end}}
+                {{if eq .Type "Ticket"}}
+                    {{range $i, $v :=  .SpendingTxns}}
```
Comment (chappjc) [inline views/tx.tmpl]: Don't range.  You only need the spending transaction hash for the first output.

Comment (McEdward) [reply views/tx.tmpl]: Alright, that helps alot

File: views/tx.tmpl [hunk 2]
```diff
@@ -32,6 +32,26 @@
...
+                    {{range .Vin}}
+                        {{if .Txid}}
+                            <tr>
+                                <td width="90" class="text-right pr-2 h1rem p03rem0 xs-w91">TICKET SPENT</td>
+                                <td><a href="/explorer/tx/{{.Txid}}">{{.Txid}}</a></td>
+                            </tr>
+                        {{end}}
+                    {{end}}
+                {{end}}
+                {{if eq .Type "Ticket"}}
```
Comment (raedah) [inline views/tx.tmpl]: @chappjc will this convert to sql on the backend, or is the db returning the whole table?

Comment (chappjc) [reply views/tx.tmpl]: The handler for the transactions route will lookup the spending transactions from sql.

**Comment by chappjc** [comment]
@McEdward Is this good to go then.  Let me know if I can retest and merge.

**Comment by McEdward** [comment]
@chappjc , yes you can retest and merge

**Pull request merged**